.gitlab/dev_container_deploy/e2e.yml

```diff
@@ -3,12 +3,8 @@ qa_agent:
   extends: .docker_publish_job_definition
   stage: dev_container_deploy
   rules:
-    - !reference [.on_aml_or_e2e_changes]
-    - !reference [.on_apm_or_e2e_changes]
-    - !reference [.on_container_or_e2e_changes]
-    - !reference [.on_npm_or_e2e_changes]
-    - !reference [.on_process_or_e2e_changes]
-    - !reference [.manual]
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As discussed I would skip it on `.if_disable_e2e_tests`

Reply:
I agree! Updated